lib/hcaptcha.rb

```diff
@@ -0,0 +1,47 @@
+require "rest_client"
+
+class HCaptcha
+  class << self
+    attr_accessor :endpoint, :secret, :site_key
+  end
```

Comment:
I think you can use `cattr_accessor` for these?

Reply:
I usually use class instance variables instead of class variables as it's tricky to debug when we do subclasses.